reinforcement_learning/include/err_constants.h

```diff
@@ -67,7 +69,9 @@ namespace reinforcement_learning { namespace error_code {
   char const * const eventhub_http_generic_s    = "http error while connecting to event hub. ";
   char const * const json_parse_error_s         = "Unable to parse JSON. ";
   char const * const invalid_argument_s         = "Invalid Argument: ";
-  char const * const exception_during_http_req_s = "http request excepton. ";
+  char const * const exception_during_http_req_s = "http request exception. ";
   char const * const eh_connstr_parse_error_s = "Unable to parse event hub connection connection string.";
+  char const * const unhandled_background_error_occurred_s = "A background thread encountered an error but there was no error handler registered. Register an error handler to see the error code and message.";
+  char const * const thread_unresponsive_timeout_s = "A background thread exceeded the watchdog timer.";
```

Comment:
If you can output this error then you can output the exception and message. Why not do this, regardless of registering an exception handler?

Reply:
After clarification from Jack, the error string makes more sense.

---
In reply to: [215702260](https://github.com/JohnLangford/vowpal_wabbit/pull/1583#discussion_r215702260) [](ancestors = 215702260)